all-repos.yaml

```diff
@@ -25,6 +25,7 @@
 - https://github.com/asottile/dead
 - https://github.com/asottile/setup-cfg-fmt
 - https://github.com/asottile/cheetah_lint
+- https://github.com/hadolint/hadolint
```

Comment:
the list is in order by appearance so this should go to the end

Reply:
Ok.  I didn't want people to get confused and use the older third-party hook implementation when there is now one on the official repository.

I'll update the order to move to the end.